ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -203,7 +203,9 @@
         "UserRoles": [
           "[LASOLICITOR]",
           "[EPSMANAGING]",
-          "[LAMANAGING]"
+          "[LAMANAGING]",
+          "[SOLICITORA]",
+          "[CHILDSOLICITORA]"
```

Comment:
I only put the permissions for these 2 because these are the only 2 roles that are assigned depending on if the applicant is a respondent or child solicitor